_posts/16-10-01-Books.md

```diff
@@ -19,6 +19,7 @@ for modern, secure, and fast cryptography.
 
 ### Paid Books
 
+* [PHP&MYSQL](https://phpandmysql.com/) - PHP book with excellent illustrations that covers all the fundamentals of PHP and MYSQL with practical examples.
```

Comment:
one small nitpick, but everywhere on the site it is stylized as `PHP & MySQL` so can we update the link to: 

```suggestion
* [PHP & MySQL](https://phpandmysql.com/) - PHP book with excellent illustrations that covers all the fundamentals of PHP and MySQL with practical examples.
```